js/freewheel.js

```diff
@@ -75,6 +75,7 @@ OO.Ads.manager(function(_, $) {
     this.initialize = function(amcIn) {
       amc = amcIn;
       amc.addPlayerListener(amc.EVENTS.INITIAL_PLAY_REQUESTED, _.bind(onPlayRequested, this));
+      amc.addPlayerListener(amc.EVENTS.REPLAY_REQUESTED, _.bind(onRePlayRequested, this));
```

Comment:
I'm going to be 'that guy'. Since 'replay' is one word can you please rename it to onReplayRequested. It'll be consistent with our other files.